src/AutoRest.CSharp/Common/Output/Models/Shared/Parameter.cs

```diff
@@ -18,6 +18,11 @@ internal record Parameter(string Name, string? Description, CSharpType Type, Con
         public FormattableString? FormattableDescription => Description is null ? (FormattableString?)null : $"{Description}";
         public CSharpAttribute[] Attributes { get; init; } = Array.Empty<CSharpAttribute>();
         public bool IsOptionalInSignature => DefaultValue != null;
+        /// <summary>
+        /// Expression to refer to the parameter under different. It could be name of the parameter, or "optiona.{Name}" depends on the context.
+        /// This is used for expanded parameters.
+        /// </summary>
+        public string NameExpression => (IsPropertyBag ? $"options.{Name.FirstCharToUpperCase()}" : Name);
```

Comment:
maybe we could remove the redundant parenthesis

Reply:
done